tests/ui/precondition-checks/read.rs

```diff
@@ -2,7 +2,7 @@
 //@ compile-flags: -Copt-level=3 -Cdebug-assertions=no -Zub-checks=yes
 //@ error-pattern: unsafe precondition(s) violated: ptr::read requires
 //@ revisions: null misaligned
-//@ ignore-test
+//@ ignore-test (unimplemented)
```

Comment:
This is wrong. The checks are implemented, they just require debug-assertions to be enabled in the sysroot.